book/graphics.md

```diff
@@ -240,15 +240,21 @@ every letter in the same place, so they all overlap! Let's fix that:
 
 ``` {.python expected=False}
 HSTEP, VSTEP = 13, 18
-x, y = HSTEP, VSTEP
+row, col = VSTEP, HSTEP
```

Comment:
Row and col seem odd because they refer to tables. How about cursor_x and cursor_y?